lightning/src/ln/channelmanager.rs

```diff
@@ -2362,6 +2410,37 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		return self.send_payment_internal(route, payment_hash, &payment_secret, None, Some(payment_id), Some(total_msat)).map(|_| ())
 	}
 
+	/// Signals that no further retries for the given payment will occur.
+	///
+	/// After this method returns, any future calls to [`retry_payment`] for the given `payment_id`
+	/// will fail with [`PaymentSendFailure::ParameterError`]. If no such event has been generated,
+	/// an [`Event::PaymentFailed`] event will be generated as soon as there are no remaining
+	/// pending HTLCs for this payment.
+	///
+	/// Note that calling this method does *not* prevent a payment from succeeding. You must still
+	/// wait until you receive either a [`Event::PaymentFailed`] or [`Event::PaymentSent`] event to
+	/// determine the ultimate status of a payment.
+	///
+	/// [`retry_payment`]: Self::retry_payment
```

Comment:
Could you similarly mention and link `abandon_payment` in `retry_payment`'s docs?